### PR TITLE
Add Portugal subdivision aliases

### PR DESCRIPTION
**Pull request by dgomes**
## Proposed change

Adds alias to each of the subdivisions (districts) in accordance with ISO 3166-2:PT

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/pin/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new `holidays` functionality in general)

## Checklist

- [ ] I've followed the [contributing guidelines][contributing-guidelines]
- [x] I've successfully run `make check`, all checks and tests are green

[contributing-guidelines]: https://github.com/vacanza/holidays/blob/dev/CONTRIBUTING.rst
[docs]: https://github.com/vacanza/holidays/tree/dev/docs/source

### CODE REVIEW

**Comment by dgomes** [comment]
tests pass locally

**Review by PPsyrius — DISMISSED** [review]
LGTM 🇵🇹

**Review by KJhellico — DISMISSED** [review]
LGTM.

File: README.rst [hunk 1]
```diff
@@ -757,7 +757,7 @@ All other default values are highlighted with bold:
      -
    * - Portugal
      - PT
-     - Districts: 01, 02, 03, 04, 05, 06, 07, 08, 09, 10, 11, 12, 13, 14, 15, 16, 17, 18, 20, 30
+     - Districts: 01 (Aveiro), 02 (Beja), 03 (Braga), 04 (Bragança), 05 (Castelo Branco), 06 (Coimbra), 07 (Évora), 08 (Faro), 09 (Guarda), 10 (Leiria), 11 (Lisboa), 12 (Portalegre), 13 (Porto), 14 (Santarém), 15 (Setúbal), 16 (Viana do Castelo), 17 (Vila Real), 18 (Viseu), 20 (Região Autónoma da Madeira), 30 (Região Autónoma dos Açores)
```
Comment (arkid15r) [inline README.rst:760]:
```suggestion
     - Districts: 01 (Aveiro), 02 (Beja), 03 (Braga), 04 (Bragança), 05 (Castelo Branco), 06 (Coimbra), 07 (Évora), 08 (Faro), 09 (Guarda), 10 (Leiria), 11 (Lisboa), 12 (Portalegre), 13 (Porto), 14 (Santarém), 15 (Setúbal), 16 (Viana do Castelo), 17 (Vila Real), 18 (Viseu), 20 (Região Autónoma dos Açores), 30 (Região Autónoma da Madeira)
```

File: holidays/countries/portugal.py [hunk 2]
```diff
@@ -63,6 +63,28 @@ class Portugal(HolidayBase, ChristianHolidays, InternationalHolidays):
...
+        "Lisboa": "11",
+        "Portalegre": "12",
+        "Porto": "13",
+        "Santarém": "14",
+        "Setúbal": "15",
+        "Viana do Castelo": "16",
+        "Vila Real": "17",
+        "Viseu": "18",
+        "Região Autónoma da Madeira": "20",
+        "Região Autónoma dos Açores": "30",
```
Comment (arkid15r) [inline holidays/countries/portugal.py:86]:
```suggestion
        "Região Autónoma dos Açores": "20",
        "Região Autónoma da Madeira": "30",
```

**Review by arkid15r — COMMENTED** [review]
Hi @dgomes thanks for the contribution!

Could you double check PT-20/PT-30 mapping? Thank you!

**Review by arkid15r — COMMENTED** [review]
Thanks for adding the aliases, @dgomes 

We also have the [AUTHORS](https://github.com/vacanza/holidays/blob/0e05a2a8dc63ff008ea1d249b09253532d215d73/AUTHORS) file if you're interested.

LGTM

**Comment by dgomes** [comment]
> Thanks for adding the aliases, @dgomes
> 
> We also have the [AUTHORS](https://github.com/vacanza/holidays/blob/0e05a2a8dc63ff008ea1d249b09253532d215d73/AUTHORS) file if you're interested.
> 
> LGTM

Thank you @arkid15r , maybe when I get the opportunity I try to address https://github.com/vacanza/holidays/issues/331 and then I can add myself to AUTHORS ;)

**Review by arkid15r — APPROVED** [review]

**Pull request merged**